src/main/java/web/spring_boot/springbootproject/controllers/mappers/MapperFileDtoToObject.java

```diff
@@ -0,0 +1,33 @@
+package web.spring_boot.springbootproject.controllers.mappers;
+
+import org.springframework.stereotype.Component;
+import web.spring_boot.springbootproject.dto.FileDto;
+import web.spring_boot.springbootproject.entity.CommentReview;
+import web.spring_boot.springbootproject.entity.ProductReview;
+import web.spring_boot.springbootproject.entity.UserReview;
+
+@Component
+public class MapperFileDtoToObject {
```

Comment:
```suggestion
public class FileMapper {
```
решту інфи буде зрозуміло з методів ( mapToDto(), etc)